src/analysis/Environments/Environments.ts

```diff
@@ -478,6 +477,7 @@ export const Environments = (props: EnvironmentsProps): ReactNode => {
       onDismiss: () => setDeleteDiskId(undefined),
       onSuccess: () => {
         setDeleteDiskId(undefined);
+        refreshWorkspaces();
```

Comment:
Refresh workspaces after delete disk

Reply:
this and the other insertions of refreshWorkspaces call just before loadData call do not appear to guarantee sequential execution.  loadData assumes refreshWorkspaces call has completed, and that cant happen unless we await refreshWorkspaces.

Reply:
Thanks! I changed it to await the calls to refreshWorkspaces and loadData.